.gitlab-ci.yml

```diff
@@ -171,3 +172,28 @@ force_release_prod:
     RELEASE_COMMAND: scripts/release.sh staging prod --force
   only:
     - staging
+
+verify_sync_partial:on-schedule:
+  stage: verify_sync
+  only:
+    refs:
+      - schedules
+  script:
+    - FILES_LAST_VERIFIED="$(python scripts/dss-ops.py secrets get --secret-names dcp/dss/${DSS_DEPLOYMENT_STAGE}/files_last_verified | jq -r '.[]')"
```

Comment:
can we change the secret name to `file_last_verified_time` or something similar? also these secrets need to be generated off the bad for a new instance of the DSS that should be reflected in the README, or there should be some way to create the secrets required if they dont exists.

Reply:
> also these secrets need to be generated off the bad for a new instance of the DSS that should be reflected in the README, or there should be some way to create the secrets required if they dont exists.

Would the appropriate way to approach this to be to set these parameters with Terraform?

Reply:
Terraform might be a good location to house this information, but i think we would need to let Terraform know that we just need to instantiate these variables, and not care about the changes that are made to them, otherwise the change the value may cause breaks in the deployment scripting. 

We might be able to do this by using `ignore_changes` by Terraform https://www.terraform.io/docs/configuration/resources.html#ignore_changes